src/main/webapp/js/userMap.js

```diff
@@ -1,13 +1,19 @@
+var countryCoordinates;
```

Comment:
Two requests are done. Result of the one of them is stored in countryCoordinates. It can be avoided only after moving all country-related data into one file